portfolio/src/main/webapp/script.js

```diff
@@ -50,8 +50,14 @@ function showFourImages() {
 }
 
 function retrieveComments() {
-    fetch('/data').then(response => response.text()).then((message) => {
+    fetch('/data?' + 'comment-count=' + document.getElementById("comment-count").value).then(response => response.text()).then((message) => {
```

Comment:
You should be able to use Template literals here: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals. This will remove the need for string concatenation.

Something like:
``"fetch(`/data?comment-count=${document.getElementById("comment-count").value)}`)"``